README.md

```diff
@@ -1 +1,70 @@
-# hello-rails-front-end
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
```

Comment:
- [ ] Good job 👏 But It looks like you have not used the [README template](https://github.com/microverseinc/readme-template) provided by Microverse. Currently, It contains content that doesn't look professional. Please use the README template provided in the [link here](https://github.com/microverseinc/readme-template) and then customize it to meet your project needs 😃 
⚠️ You will notice that It is an Updated template, a lot new sections and features has been added so please follow it all 😃